src/game_battlealgorithm.cpp

```diff
@@ -1094,20 +1102,45 @@ std::string Game_BattleAlgorithm::Skill::GetStartMessage() const {
 		if (Player::IsRPG2kE()) {
 			auto* target = GetTarget();
 			return Utils::ReplacePlaceholders(
-				skill.using_message1 + '\n' + skill.using_message2,
+				skill.using_message1,
 				{'S', 'O', 'U'},
 				{GetSource()->GetName(), (target ? target->GetName() : "???"), skill.name}
 			);
 		}
 		else {
-			return source->GetName() + skill.using_message1 + '\n' + skill.using_message2;
+			return source->GetName() + skill.using_message1;
 		}
 	}
 	else {
 		return skill.name;
 	}
 }
 
+bool Game_BattleAlgorithm::Skill::IsSecondStartMessage() const {
+	return Player::IsRPG2k() && (!item || item->using_message != 0) && !skill.using_message2.empty();
```

Comment:
> 2.5: RM2000: If the second line of a Skill is not empty

Grammar wise the name should be "HasSecondStartMessage".
This suggestion is maybe minimal more inefficient but the whole function could be removed and the code in battle_rpg2k.cpp replaced with an "empty()" check.